### ISSUE TITLE
Doctrine ODM Support

### ISSUE THREAD

**Issue by isaachbn** [marcojanssen/silex-rest#16]
You think in adding support Doctrine ODM?

**Comment by marcojanssen** [comment]
It should be pretty easy to do, considering this is just a skeleton i don't see why i should add it here though..

**Comment by isaachbn** [comment]
There is some considerable dependencies such as:  RestEntityService usage $this->app['orm.em'],  ResolverService are not using ObjectManager.

**Comment by marcojanssen** [comment]
Yeah you're right, probably don't have the time to look into it atm but will keep the issue open as a reminder
